plugins/modules/ec2_vpc_vpn.py

```diff
@@ -46,6 +46,11 @@
     default: False
     type: bool
     required: false
+  transit_gateway_id:
+    description:
+      - The ID of the transit gateway. If you specify a transit gateway, you cannot specify a virtual private gateway.
```

Comment:
```suggestion
      - The ID of the transit gateway.
      - Mutually exclusive with I(vpn_gateway_id).
```

A similar line needs to be added to the docs for vpn_gateway_id, mentioning that it's mutually exclusive with transit_gateway_id.